containerd/socket-tracer/containerd-socket-tracer.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: containerd-socket-tracer
+  namespace: default
+  labels:
+    k8s-app: containerd-socket-tracer
+spec:
+  selector:
+    matchLabels:
+      app: containerd-socket-tracer
+  template:
+    metadata:
+      labels:
+        app: containerd-socket-tracer
+    spec:
+      hostPID: true
+      hostNetwork: true
+      containers:
+      - name: tracer
+        image: ubuntu:25.04
+        command: ["/bin/bash", "-c"]
+        args:
+        - |
+          apt-get update && \
+          apt-get install -y bpftrace jq && \
+          /scripts/trace.sh
```

Comment:
if this needs to start very early, we may consider packing the script right in this spec. Downloading ConfigMap may slow down the bootstrap.